helm/charts/example-idp/README.md

```diff
@@ -6,30 +6,31 @@ A Helm chart for deploying the reference implementation for the User Login and C
 
 ## Values
 
-| Key | Type | Default | Description |
-|-----|------|---------|-------------|
-| affinity | object | `{}` |  |
-| deployment | object | `{"extraEnv":[]}` | Set of values for the app deployment |
-| deployment.extraEnv | list | `[]` | Array of extra Envs to be added to the deployment. K8s format expected - name: FOO   value: BAR |
-| fullnameOverride | string | `""` |  |
-| image.pullPolicy | string | `"IfNotPresent"` |  |
-| image.repository | string | `"oryd/hydra-login-consent-node"` |  |
-| image.tag | string | `"v1.4.6"` |  |
-| imagePullSecrets | list | `[]` |  |
-| ingress.annotations | object | `{}` |  |
-| ingress.className | string | `""` |  |
-| ingress.enabled | bool | `false` |  |
-| ingress.hosts[0].host | string | `"example-idp.localhost"` |  |
-| ingress.hosts[0].paths[0].path | string | `"/"` |  |
-| ingress.hosts[0].paths[0].pathType | string | `"ImplementationSpecific"` |  |
-| ingress.tls | list | `[]` |  |
-| nameOverride | string | `""` |  |
-| nodeSelector | object | `{}` |  |
-| resources | object | `{}` |  |
-| service.annotations | object | `{}` | If you do want to specify annotations, uncomment the following lines, adjust them as necessary, and remove the curly braces after 'annotations:'. |
-| service.port | int | `3000` |  |
-| service.type | string | `"ClusterIP"` |  |
-| tolerations | list | `[]` |  |
+| Key                                | Type   | Default                           | Description                                                                                                                                       |
```

Comment:
Those files are auto-generated by helm-docs plugin from our values.yaml. The `make format` and the plugin will override each other all the time 😞

Reply:
Thanks for pointing this out to me. Prettier is ignoring the entire `helm/charts` folder now since all files in it seem auto-generated.